package.json

```diff
@@ -21,6 +21,7 @@
     "clean-cra-sw": "rm -f build/precache-manifest.*.js && rm -f build/service-worker.js",
     "build": "react-scripts build && npm run build-sw && npm run clean-cra-sw",
     "test": "react-scripts test --env=jest-environment-jsdom-sixteen",
+    "test:coverage": "CI=true react-scripts test --env=jsdom",
```

Comment:
yay! Test coverage tracking is the best!

Reply:
It seems like wizardry a little bit. Stoked that we're analyzing our coverage in this way!